nano/node/distributed_work.cpp

```diff
@@ -448,6 +451,23 @@ void nano::distributed_work_factory::cleanup_finished ()
 	}
 }
 
+void nano::distributed_work_factory::stop ()
+{
```

Comment:
Should `items` be cleared at the end of this function? Otherwise in the destructor they can be cancelled again

Reply:
Each vector of work is erased in `cancel()` , but I've added a clear to make sure, and a `stopped` flag to avoid calling `stop()` twice or adding new work in `make ()` after stopping